yasa/hypno.py

```diff
@@ -391,6 +404,14 @@ def scorer(self):
         """The scorer name."""
         return self._scorer
 
+    @property
+    def proba(self):
+        """
+        If specified, a :py:class:`pandas.DataFrame` with the probability of each sleep stage
+        for each epoch in hypnogram.
+        """
+        return self._proba
+
     # CLASS METHODS BELOW
 
     def as_annotations(self):
```

Comment:
The MNE annotations glossary link in the docstrings is dead now. And that actually highlights an anxiety of mine about this nice and convenient `as_annotations` method; I really like using the BIDS standards for events files, but to-date I've struggled with comprehending the terminology in MNE between `events` and `annotations`. They have `events_to_annotations` and `annotations_to_events` methods, and I'm sure that if I look at the docs today there will be a clear differentiation, but as I recall I am consistently "confident-and-subsequently-confused" about how they handle this.

So I think this method is useful but I don't know about the name of it. Maybe it's just `as_events` for now? Because that is also consistent with BIDS, which is just looking for an events dataframe, and that's what this is.

Reply:
Agreed — renamed to `as_events`